src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -88,16 +91,15 @@ async def gen_pos_stance_messages(
                 )
             )
             if str(orders_decided) != "FCT ()":
-                for pow in self.stance.stance[self.power_name]:
+                for pow in self.allies:
                     if (
                         pow != self.power_name
-                        and self.stance.stance[self.power_name][pow] > 0
+                        and pow in self.allies
```

Comment:
redundant line

Reply:
oh I just changed according to the prev version but yeah let's remove the redundant

Reply:
done